tasks/modules.py

```diff
@@ -130,11 +130,11 @@ def dependency_path(self, agent_version):
         ".",
         targets=["./pkg", "./cmd", "./comp"],
     ),
-    "cmd/agent/common/path": GoModule("cmd/agent/common/path", independent=True),
-    "comp/api/api/def": GoModule("comp/api/api/def", independent=True),
+    "cmd/agent/common/path": GoModule("cmd/agent/common/path", independent=True, used_by_otel=True),
```

Comment:
Is this change expected?

Reply:
`cmd/agent/common/path` is used in `comp/core/config` module.
It is only used in `comp/core/config/params_test.go` to get `DefaultConfPath`.
Should we try to use another way to get `DefaultConfPath` to avoid having it as a dependency ?